common/changes/@microsoft.azure/openapi-validator-rulesets/akhilailla-ListBySubscriptionForExtensionResourceFalsePositive_2023-08-22-22-01.json

```
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft.azure/openapi-validator-rulesets",
+      "comment": "update extensionResourceRegex to also include unqualified paths",
```

I think this might be clearer from the user perspective. Is this accurate for the change?
```suggestion
      "comment": "Add support for extension resource paths that are not fully qualified",
```